js/src/ui/Form/TypedInput/typedInput.js

```diff
@@ -236,17 +238,30 @@ export default class TypedInput extends Component {
     );
   }
 
+  getNumberValue (value) {
+    if (!value) {
+      return value;
+    }
+
+    const rawValue = typeof value === 'string'
+      ? value.replace(/,/g, '')
+      : value;
+
+    return new BigNumber(rawValue);
+  }
+
   renderInteger (value = this.props.value, onChange = this.onChange) {
-    const { allowCopy, label, error, hint, min, max, readOnly } = this.props;
+    const { allowCopy, className, label, error, hint, min, max, readOnly } = this.props;
     const param = this.getParam();
 
     const realValue = value
-      ? (new BigNumber(value))[readOnly ? 'toFormat' : 'toNumber']()
+      ? this.getNumberValue(value)[readOnly ? 'toFormat' : 'toNumber']()
```

Comment:
This will probably break when `getNumberValue` returns from the first test, i.e. where it is null or undefined